src/java/Logica/Entidades/Grupo.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package Logica.Entidades;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Grupo implements Serializable {
+    
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Integer id;
+    private String nombre;
+    private String tema;
+    private List<Integer> miembros;
```

Comment:
@Bmorseletto la lista de miembros deberia ser una List<Lector>. Se podria usar la notacion @OneToMany de JPA para indicar que un grupo puede tener muchos lectores. Ver referencia: https://www.geeksforgeeks.org/jpa-one-to-many-mapping/